app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car_list: list) -> float:
+        washed_cars = [
+            car for car in car_list if car.clean_mark < self.clean_power]
+        income = 0
+        for car in washed_cars:
+            price = car.comfort_class * (self.clean_power - car.clean_mark)
+            price *= self.average_rating / self.distance_from_city_center
```

Comment:
Make a call to `wash_single_car` in this loop, currently, you do not evoke this method anywhere.